consent-ui/src/app/common/session.service.ts

```diff
@@ -9,6 +9,14 @@ export class SessionService {
   public setTTL(authorizationId: string, ttl: string) {
     sessionStorage.setItem(authorizationId + Session.COOKIE_TTL, ttl);
   }
+
+  public isLongTimeCookie() {
+    return sessionStorage.getItem(Session.LONG_LIFE_COOKIE);
```

Comment:
Session.LONG_LIFE_COOKIE should be prefixed with `authorizationId` like in `setTTL`, because user may have multiple authorization sessions

Reply:
Yeah you are right